src/explorer/SiteTreeItem.ts

```diff
@@ -110,6 +120,12 @@ export abstract class SiteTreeItem implements IAzureParentTreeItem {
     public async editScmType(client: WebSiteManagementClient): Promise<string> {
         return await this.siteWrapper.editScmType(client);
     }
+
+    private async reloadSite(client: WebSiteManagementClient): Promise<void> {
```

Comment:
I think the start/stop functions use the Azure API to confirm that the web app has started/stopped.  Do we need to reacquire the resource through the API to update (Running)/(Stopped) and reconstruct the SiteWrapper to confirm the state?

Reply:
Good point! I'll make a bit of adjustment...

Reply:
No need to reload the site or reconstruct the sitewrapper. Just use the existing `siteWrapper.getState` method